scripts/promise_batch_imports.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+import requests
+import logging
+
+from infogami import config  # noqa: F401
+from openlibrary.config import load_config
+from openlibrary.core.imports import Batch
+from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
+
+logger = logging.getLogger("openlibrary.importer.promises")
+
+
+def format_date(date: str) -> str:
+    y = date[0:4]
+    m = date[4:6]
+    d = date[6:8]
+    return f"{y}-{m}-{d}"
+
+
+def map_book_to_olbook(book, promise_id):
+    asin_is_isbn_10 = book.get('ASIN') and book.get('ASIN')[0].isdigit()
+    publish_date = book['ProductJSON'].get('PublicationDate')
+    title = book['ProductJSON'].get('Title')
```

Comment:
A small number of records do not have titles, and are failing validation.  Is there an alternate field that contains the title that can be used as a failback?